diff --git a/node_modules/undici/index.js b/node_modules/undici/index.js
index 708a8ee..f4d75ca 100644
--- a/node_modules/undici/index.js
+++ b/node_modules/undici/index.js
@@ -59,8 +59,8 @@ module.exports.cacheStores = {
   MemoryCacheStore: require('./lib/cache/memory-cache-store')
 }
 
-const SqliteCacheStore = require('./lib/cache/sqlite-cache-store')
-module.exports.cacheStores.SqliteCacheStore = SqliteCacheStore
+// const SqliteCacheStore = require('./lib/cache/sqlite-cache-store')
+module.exports.cacheStores.SqliteCacheStore = null
 
 module.exports.buildConnector = buildConnector
 module.exports.errors = errors
diff --git a/node_modules/undici/lib/util/runtime-features.js b/node_modules/undici/lib/util/runtime-features.js
index 3e62dc0..b927ace 100644
--- a/node_modules/undici/lib/util/runtime-features.js
+++ b/node_modules/undici/lib/util/runtime-features.js
@@ -6,7 +6,7 @@
 const lazyLoaders = {
   __proto__: null,
   'node:crypto': () => require('node:crypto'),
-  'node:sqlite': () => require('node:sqlite'),
+  'node:sqlite': () => { throw new Error('node:sqlite is not supported') },
   'node:worker_threads': () => require('node:worker_threads'),
   'node:zlib': () => require('node:zlib')
 }
